js/main.js

```diff
@@ -0,0 +1,84 @@
+let numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+const typeHousings = ["palace", "flat", "house", "bungalow", "hotel"];
+const randomText = [
+	"произвольный заголовок",
+	"Произвольный текст",
+	"Lorem ipsum",
+];
+const houseFeatures = [
+	"wifi",
+	"dishwasher",
+	"parking",
+	"washer",
+	"elevator",
+	"conditioner",
+];
+const LAT_MIN = 35.65;
+const LAT_MAX = 35.7;
+const LNG_MIN = 139.7; //
+const LNG_MAX = 139.8;
+const checkInOutData = ["12:00", "13:00", "14:00"];
+const photos = [
+	"https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg",
+	"https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg",
+	"https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg",
+];
+
+const getRandomNumber = (min, max, floatingNumber) => {
+	if (floatingNumber) {
+		let res = Math.random() * (max - min) + min;
+		res = res.toFixed(5);
+		return parseFloat(res);

```

Comment:
только здесь тогда не toFixed(5) а toFixed(floatingNumber)
```suggestion
const getRandomNumber = (min, max, floatingNumber) => {
	if (floatingNumber) {
		let res = Math.random() * (max - min) + min;
		res = res.toFixed(floatingNumber);
		return parseFloat(res);

```